scripts/lib/CIME/utils.py

```diff
@@ -254,6 +254,27 @@ def get_cime_root(case=None):
     logger.debug( "CIMEROOT is " + cimeroot)
     return cimeroot
 
+def get_src_root(case=None):
+    """
+    Return the absolute path to the root of SRCROOT.
```

Comment:
Currently this just seems to assume that srcroot is one level up from cimeroot. Can you explain your vision for what this function will become?

Reply:
My vision is if cime_model is cesm and the share directory is in cimeroot, then srcroot=cimeroot.  This would be for a cime stand alone clone.  Otherwise, srcroot=../cimeroot.